src/PaymentManager/Payment/PayPalSmartPaymentButton.php

```diff
@@ -107,6 +107,8 @@ public function getName()
      *
      * @return mixed - either an url for a link the user has to follow to (e.g. paypal) or
      *                 an symfony form builder which needs to submitted (e.g. datatrans and wirecard)
+     *
+     * @deprecated since it was removed from Pimcore 10. This method will be renamed in pimcore/payment-provider-paypal-smart-payment-button 2.0, please see PR #16.
```

Comment:
we should provide replacement already in this version otherwise deprecation has no meaning to it.